pyiron_base/jobs/master/list.py

```diff
@@ -24,108 +25,31 @@
 __date__ = "Sep 1, 2017"
 
 
+# Modular Docstrings
+_doc_str_list_master_attr = (
+    _doc_str_generic_master_attr
+    + "\n"
+    + """\
+        .. attribute:: submission_status
+
+            Monitors how many jobs have been submitted and how many have to be submitted in future
+"""
```

Comment:
```suggestion
    # Attributes:
    + """\
        submission_status (SubmissionStatus): Monitors how many jobs have been submitted and how many have to be submitted in future
"""
```

e.g. for google conversion. Not sure the indentation lines up and black might want to break the line, but you get the idea.